tests/test_ApproximateBandpasses.py

```diff
@@ -0,0 +1,75 @@
+"""
+Module to test that Approximate Bandpasses are in sync with the official LSST
+Bandpasses from SYSENG
+
+Note: While the LSST bandpasses list throughput values corresponding to
+wavelengths in the range of 300.0-1150.0 nm, the `approximate_baseline`
+directory of throughputs is created by a script manually. It is thus possible
+for this directory to fall out of sync with the SYSENG values in `baseline`.
+This module is intended to test whether this is happening.
+"""
+from __future__ import with_statement
+import os
+import unittest
+import numpy as np
+import lsst.utils.tests
+from lsst.utils import getPackageDir
+from lsst.sims.photUtils import BandpassDict
+
+def setup_module(module):
+    lsst.utils.tests.init()
+
+class ApproximateBandPassTest(unittest.TestCase):
+    """
+    Tests for the approximate Bandpasses in the throughputs directory
+    """
+    
```

Comment:
If you set `longMessage = True` here (in the class definition but outside of any method), then `unittest` will append your `msg` kwargs to whatever error message it naturally wants to emit.  I think that would be helpful, as it would provide us with both why `unittest` has declared a failure and your interpretation of what that failure means.